### ISSUE TITLE
CellBender scRNA-seq tutorial

### ISSUE THREAD

**Issue by giorgiatosoni** [mousepixels/sanbomics_scripts#10]
Hi,

Hi, I was trying to follow the complete scRNA-seq tutorial using CellBender h5 out file, by changing adata = anndata_from_h5(mtx_path) (anndata_fromh5 being https://github.com/broadinstitute/CellBender/issues/57).
When I train the solo model I noticed something weird, it looks like it's not working:

/home/gtosoni/miniconda3/envs/scvi-env/lib/python3.9/site-packages/anndata/_core/anndata.py:1830: UserWarning: Variable names are not unique. To make them unique, call `.var_names_make_unique`.
  utils.warn_names_duplicates("var")
/home/gtosoni/miniconda3/envs/scvi-env/lib/python3.9/site-packages/scvi/model/_utils.py:287: UserWarning: This dataset has some empty cells, this might fail inference.Data should be filtered with `scanpy.pp.filter_cells()`
  warnings.warn(
GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Epoch 267/267: 100%|██████████| 267/267 [13:27<00:00,  2.96s/it, loss=478, v_num=1]

**INFO     Creating doublets, preparing SOLO model.                                            
/home/gtosoni/miniconda3/envs/scvi-env/lib/python3.9/site-packages/scvi/external/solo/_model.py:185: RuntimeWarning: divide by zero encountered in log
  latent_adata = AnnData(np.concatenate([latent_rep, np.log(lib_size)], axis=1))
/home/gtosoni/miniconda3/envs/scvi-env/lib/python3.9/site-packages/anndata/_core/anndata.py:1785: FutureWarning: X.dtype being converted to np.float32 from float64. In the next version of anndata (0.9) conversion will not be automatic. Pass dtype explicitly to avoid this warning. Pass `AnnData(X, dtype=X.dtype, ...)` to get the future behavour.
  [AnnData(sparse.csr_matrix(a.shape), obs=a.obs) for a in all_adatas],
GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Epoch 1/400:   0%|          | 1/400 [00:01<08:54,  1.34s/it, loss=nan, v_num=1]
Monitored metric validation_loss = nan is not finite. Previous best value was inf. Signaling Trainer to stop.**

Any idea why is that? Thank you!!
